python/setup.py

```diff
@@ -0,0 +1,51 @@
+from setuptools import setup, find_packages
+
+from setuptools import Extension
+from setuptools.command.build_ext import build_ext
+import sys
+import setuptools
+
+import glob
+import os
+
+ext_modules = [
+    Extension(
+        'libsunbeam',
+        [
+                'cxx/connection.cpp',
+                'cxx/deck.cpp',
+                'cxx/deck_keyword.cpp',
+                'cxx/eclipse_3d_properties.cpp',
+                'cxx/eclipse_config.cpp',
+                'cxx/eclipse_grid.cpp',
+                'cxx/eclipse_state.cpp',
+                'cxx/group.cpp',
+                'cxx/group_tree.cpp',
+                'cxx/parser.cpp',
+                'cxx/schedule.cpp',
+                'cxx/sunbeam_state.cpp',
+                'cxx/table_manager.cpp',
+                'cxx/well.cpp',
+                'cxx/sunbeam.cpp'
+        ],
+        libraries=['opmcommon', 'boost_filesystem', 'boost_regex', 'ecl'],
+        language='c++',
+        undef_macros=["NDEBUG"],
+        include_dirs=["pybind11/include"]
+    ),
+]
+
+setup(
+    name='Sunbeam',
+    package_dir = {'': 'python'},
+    packages=[
+                'sunbeam',
+                'sunbeam.tools',
+                'sunbeam.deck',
+            ],
+    ext_modules=ext_modules,
+    license='Open Source',
+    zip_safe=False,
+    tests_suite=['tests'],
```

Comment:
I think this should be: `test_suite = 'tests'` - at least I get a warning: 
```bash
/usr/lib/python2.7/distutils/dist.py:267: UserWarning: Unknown distribution option: 'tests_suite'
  warnings.warn(msg)
```
with the current

Reply:
NB - it should be `test_suite` - in *singular*